src/test_l1_tvSettings.c

```diff
@@ -2523,8 +2522,8 @@ void test_l1_tvSettings_positive_GetSupportedBacklightModes (void)
 	count = UT_KVP_PROFILE_GET_LIST_COUNT("tvsettings/BacklightControl/index")
 	for (size_t i = 0; i < count; i++)
 	{
-        snprintf(keyValue, UT_KVP_MAX_ELEMENT_SIZE, "tvSettings/BacklightControl/index/%d", j);
-        modeValue = UT_KVP_PROFILE_GET_UINT32(keyValue);
+		snprintf(keyValue, UT_KVP_MAX_ELEMENT_SIZE, "tvSettings/BacklightControl/index/%d", j);
```

Comment:
index value is read with j but loop value is assigning to i

Reply:
updated